pkg/containerd/resolver/auth.go

```diff
@@ -0,0 +1,198 @@
+/*
```

Comment:
Is this a copy/rewrite of a containerd file?

If so, please describe in a comment:
1) Which file is this copied from?
2) What is changed?
3) When can we remove the copy?